### PR TITLE
Add Tutorial for Identity Judgement

### PR DESCRIPTION
**Pull request by WRRicht3r**

### CODE REVIEW

File: src/requesting-identity-judgement.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+## TL;DR
+
+- You can set an on-chain identity to make it easier for others to recognize your address.
+- As a Validator, it adds extra confidence to Nominators if they know your on-chain identity.
+- This tutorial will help you set up an on-chain identity and request a judgement to your identity (as a trusted or verified entity on the SORA network)
+
+## 1. Setting an On-Chain Identity
+
+1.  To do this, click the 3 vertical dots next to your account on
```
Comment (outoftardis) [inline src/requesting-identity-judgement.md]:
```suggestion
1. Click the 3 vertical dots next to your account on
```

File: src/requesting-identity-judgement.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    Polkadot.js and select `Set on-chain identity`.
+
+![](.gitbook/assets/set-onchain-identity-1.png)
+
+2. Add the details you wish to be public and click `Set identity`.
+
+![](.gitbook/assets/set-onchain-identity-2.png) 3. A deposit of 0.01XOR is required.
```
Comment (outoftardis) [inline src/requesting-identity-judgement.md]:
```suggestion
![](.gitbook/assets/set-onchain-identity-2.png)

3. Deposit 0.01XOR.
```

File: src/requesting-identity-judgement.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+   100 XOR, excluding the transaction fee.
+
+![](.gitbook/assets/request-judgement-registrar-2.png)
+
+3. Submit the transaction.
+
+## 3. Donate to the Community Marketing Funds Address
+
+Donate a minimum of 5 XSTUSD per account requesting judgement to the
+Community Marketing Funds address.
```
Comment (outoftardis) [inline src/requesting-identity-judgement.md]:
```suggestion
Community Marketing Funds address:
```

File: src/requesting-identity-judgement.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+`cnSN9X2HNU3uYjRbGvTwpgUdtVDuNk1g29XkcQNoNBQxgt1ii`
+
+:::info
+This is a community base wallet which funds are used for marketing SORA.
```
Comment (outoftardis) [inline src/requesting-identity-judgement.md]:
```suggestion
The funds from this community base wallet are used for marketing SORA.
```

File: src/requesting-identity-judgement.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+:::
+
+## 4. Contact the Registrar
+
+- Direct contact with the registrar is required - the contact info is in their identity.
+  `Reg_Index 1` is `SPICY`.
+  You can contact the registrar on:
+- Telegram - [SpicyHotWings ](https://t.me/SpicyHotWings)
+- Twitter - [@mishokanaria](https://twitter.com/MishoKanaria)
+- Riot (Matrix) - [spicyhotwings](https://matrix.to/#/@spicyhotwings:matrix.org)
```
Comment (outoftardis) [inline src/requesting-identity-judgement.md]:
```suggestion
Direct contact with the registrar is required. You can find the contact info is in their identity. For example, `Reg_Index 1` is `SPICY`.

You can contact the registrar on:
- Telegram: [SpicyHotWings](https://t.me/SpicyHotWings)
- Twitter: [@mishokanaria](https://twitter.com/MishoKanaria)
- Riot (Matrix): [spicyhotwings](https://matrix.to/#/@spicyhotwings:matrix.org)
```

**Review by outoftardis — APPROVED** [review]

File: src/id.md [hunk 6]
```diff
@@ -42,11 +46,11 @@ Follow these steps to set up an identity:
 
 Once the transaction is completed your account will show an identity!
 
-## Request Identity Judgment
+## Requesting Identity Judgment
 
 Once you have finished following all the steps outlined in the [Register an identity](/id.md#register-an-identity) section, you can request an additional verification to obtain a prominent green checkmark next to your account name. This verification serves as an indication that you are a recognized and verified user, allowing others to easily identify and trust your account.
 
-1. Request judgement from a registrar on polkadot.js.
+### 1. Request judgement from a registrar on polkadot.js.
```
Comment (bgodlin) [inline src/id.md:54]: In the preceding section, the steps lack the heading decoration. Consistent formatting style needs to be applied.

**Review by bgodlin — CHANGES_REQUESTED** [review]

File: src/id.md (hunk 6 above)
Comment (WRRicht3r) [reply src/id.md:54]: The steps in the preceding section are quite short and do not have the same sub-instructions as this one. I have made the requested changes for your consideration.

**Review by bgodlin — APPROVED** [review]

**Pull request merged**